internal/driver/cli.go

```diff
@@ -58,6 +59,7 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 	flagTools := flag.String("tools", os.Getenv("PPROF_TOOLS"), "Path for object tool pathnames")
 
 	flagTimeout := flag.Int("timeout", -1, "Timeout in seconds for fetching a profile")
+	flagHTTPPort := flag.Int("http", 0, "Present interactive web based UI at the specified http port")
```

Comment:
That's what I had tried initially, but it became awkward for a few reasons:

1. The usage message for the new command showed up under `Output formats` which seemed odd.

2. Commands only support string parameters, whereas I wanted a port number.

3. Making it a command would allow `http _port_` as a command in the interactive shell, which I wasn't sure we wanted to support.

So possible, but not sure it would be better/simpler than the current approach.